tests/testthat/test-basics.R

```diff
@@ -532,3 +533,91 @@ test_that("recipe() error for table input (#1416)", {
     recipe(Titanic, Survived ~ .)
   )
 })
+
+
```

Comment:
This is where we are actually testing this behavior. All the other changes to the test files are moving the arg from `prep()` to `recipe()`